modules/zivid/point_cloud.py

```diff
@@ -0,0 +1,66 @@
+"""Contains the PointCloud class."""
+import numpy
+
+import _zivid
+
+
+class PointCloud:
+    """A point cloud."""
+
+    def __init__(self, internal_point_cloud):
+        """Create a point cloud from an internal point cloud.
+
+        Args:
+            internal_point_cloud: a internal point cloud
+
+        """
+        if not isinstance(internal_point_cloud, _zivid.PointCloud):
+            raise ValueError(
+                "Unsupported type for argument internal_point_cloud. Got {}, expected {}".format(
+                    type(internal_point_cloud), type(_zivid.PointCloud)
+                )
+            )
+        self.__impl = internal_point_cloud
+
+    def to_array(self):
+        """Convert point cloud to numpy array.
+
+        Returns:
+            a numpy array
+
+        """
+        self.__impl.assert_not_released()
+        return numpy.array(self.__impl)
+
+    @property
+    def height(self):
+        """Maximum height of point cloud.
```

Comment:
When 1:1 wrapping a function from C++ just copy the documentation.

 * Consistency
 * Time already spent on thinking about wording. No point in spending that time again.

This goes for all functions in PR and (also for the all functions in the entire wrapper, but that is not part of this PR).

Reply:
Fixed.